pallets/token-manager/src/benchmarking.rs

```diff
@@ -276,32 +282,10 @@ benchmarks! {
         lower.t1_recipient
     )
     verify {
-        assert_eq!(<T as pallet::Config>::Currency::free_balance(&lower.from_account_id), 0u32.into());
+        assert_eq!(<T as pallet::Config>::Currency::free_balance(&lower.from_account_id), 1000u32.into());
```

Comment:
This check only verifies that no tokens have been burned but doesn't guarantee that scheduling has completed successfully. Can we check for the `LowerScheduled` event as well please?